CHANGELOG.md

```diff
@@ -1,3 +1,82 @@
+<a name="v0.9.0"></a>
+## [v0.9.0](https://github.com/sourcenetwork/defradb/compare/v0.8.0...v0.9.0)
+
+> 2023-11-14
+
+DefraDB v0.9 is a major pre-production release. Until the stable version 1.0 is reached, the SemVer minor patch number will denote notable releases, which will give the project freedom to experiment and explore potentially breaking changes.
+
+To get a full outline of the changes, we invite you to review the official changelog below. This release does include a Breaking Change to existing v0.8.x databases. If you need help migrating an existing deployment, reach out at [hello@source.network](mailto:hello@source.network) or join our Discord at https://discord.gg/w7jYQVJ/.
+
+> 2024-01-15
+
+### Features
+
+* Mutation typed input ([#2167](https://github.com/sourcenetwork/defradb/issues/2167))
+* Add PN Counter CRDT type ([#2119](https://github.com/sourcenetwork/defradb/issues/2119))
+* Allow users to add Views ([#2114](https://github.com/sourcenetwork/defradb/issues/2114))
+* Add unique secondary index ([#2131](https://github.com/sourcenetwork/defradb/issues/2131))
+* New cmd for docs auto generation ([#2096](https://github.com/sourcenetwork/defradb/issues/2096))
+* Add blob scalar type ([#2091](https://github.com/sourcenetwork/defradb/issues/2091))
+
+### Fixes
+
+* Add entropy to counter CRDT type updates ([#2186](https://github.com/sourcenetwork/defradb/issues/2186))
+* Handle multiple nil values on unique indexed fields ([#2178](https://github.com/sourcenetwork/defradb/issues/2178))
+* Filtering on unique index if there is no match ([#2177](https://github.com/sourcenetwork/defradb/issues/2177))
+
+### Performance
+
+* Switch LensVM to wasmtime runtime ([#2030](https://github.com/sourcenetwork/defradb/issues/2030))
+
+### Refactoring
+
+* Add strong typing to document creation ([#2161](https://github.com/sourcenetwork/defradb/issues/2161))
+* Rename key,id,dockey to docID terminology ([#1749](https://github.com/sourcenetwork/defradb/issues/1749))
+* Simplify Merkle CRDT workflow ([#2111](https://github.com/sourcenetwork/defradb/issues/2111))
+
+### Testing
+
+* Add auto-doc generation ([#2051](https://github.com/sourcenetwork/defradb/issues/2051))
+
+### Continuous integration
```

Comment:
thought: Do we want this header in the change log?

Reply:
They were there in the past. It is not super relevant information for users so I'm open to removing it.

Reply:
Lets keep it there for this one and discuss for future releases.